### ISSUE TITLE
How to ignore comments especially jsdoc ones?

### ISSUE THREAD

**Issue by MV88** [Alex-Rudoy/custom-import-sort#4]
Currently with version 0.1.2 it sorts also comments even if there are no rule 
I have checked a bit the code and it seems to me that this is by design

here
https://github.com/Alex-Rudoy/custom-import-sort/blob/a0659a34cf9a8f777f2b7d1f5ab8cdb8847390df/src/utils/splitImportsAndNonImports.ts#L27
and here
https://github.com/Alex-Rudoy/custom-import-sort/blob/a0659a34cf9a8f777f2b7d1f5ab8cdb8847390df/src/utils/splitImportsAndNonImports.ts#L45-L48

Example 
```js
import React from "react";

export const func1....
export const func2....

/**
 * comment
 * @prop {string} filterText
*/
export const loopFilter = ({ filterText }) => {}

```

becomes 

```js
import React from "react";

/**
 * comment
 * @prop {string} filterText
*/

export const func1....
export const func2....

export const loopFilter = ({ filterText }) => {}

```
These are my settings, but I do not think they are important as this happens only if I have just the first rule in use:

```
"customImportSort.sortingSettings": [
    {
      "regex": "^react$",
      "importance": 4
    },
    {
      "regex": ".",
      "importance": 0,
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)components",
      "importance": 3
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)actions",
      "importance": 3
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)reducers",
      "importance": 3
    },
    {
      "regex": "^@",
      "importance": 1,
      "lineafter": true
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)mapstore",
      "importance": 3,
      "lineafter": true
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)js",
      "importance": 4
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)src",
      "importance": 4
    },
    {
      "regex": "^\\.\\/",
      "importance": 1
    },
    {
      "regex": "^\\.\\.\\/",
      "importance": 2,
      "lineafter": true
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)util",
      "importance": 3
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)api",
      "importance": 3
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)service",
      "importance": 3
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)store",
      "importance": 3
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)store.*\\/use",
      "importance": 4
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)hook",
      "importance": 3,
      "lineafter": true
    },
    {
      "regex": "const",
      "importance": 1
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)rout",
      "importance": 3
    },
    {
      "regex": "^(@|[\\.\\.\\/]+)typings",
      "importance": 3
    },
    {
      "regex": "\\.types?(\\.tsx?)?$",
      "importance": 1,
      "lineafter": true
    },
    {
      "regex": "^\\.?\\.\\/.*\\.types(\\.tsx?)?$",
      "importance": 4,
      "lineafter": true
    },
    {
      "regex": "\\.s?css$",
      "importance": 4,
      "lineafter": true
    }
  ],
```

even with only first rule active, the react one, it moves the comment up

Do you need e reproducible case to check?

**Comment by MV88** [comment]
Hi,
sorry to bother you again,
but is there a way to ignore the location of comments containing copyrights?
like

wrong order 

```
/**
 * copyrights
 * text
*/

import c from "c";
import a from "a";
import z from "z";

/**
 * 1
*/
const x = "t";
```

expected order with preserving comments

```
/**
 * copyrights
 * text
*/

import a from "a";
import c from "c";
import z from "z";

/**
 * 1
*/
const x = "t";
```
 

current order with command 

```
import a from "a";
import c from "c";
import z from "z";

/**
 * copyrights
 * text
*/

/**
 * 1
*/
const x = "t";

```

do I need to open a dedicated issue? 
or do you simply welcome a new PR?

**Comment by Alex-Rudoy** [comment]
I will try to fix it myself, had some thoughts last time

**Comment by Alex-Rudoy** [comment]
hi @MV88 
sorry for doing this for so long, but fixed and published v1.0.0
removed preserving comments option
now regular sorting must be working just as you need

**Comment by MV88** [comment]
wonderful great job!
I'll try out new version
I'm glad I've helped making things better :)
